js/djtilesource.js

```diff
@@ -1,92 +1,134 @@
+/*jslint browser: true*/
+/*global OpenSeadragon, Drupal*/
+/**
+ * @file
+ * Defines the source for exposing Djatoka to OpenSeadragon.
+ */
 (function($) {
+  'use strict';
+
+  // Remove IIFTileSource, as it assumes Djatoka responses belong to it for some reason.
+  $.IIIFTileSource = undefined;
```

Comment:
Walking through this a bit it appears that OpenSeadragon makes some assumptions for a IIIFTileSource. My javascript fails on the function `canBeTiled()` here https://openseadragon.github.io/docs/openseadragon.js.html#line12302 as `options.profile` is undefined.
So while the data does specify the `height`, `width` and `identifier`. This function crashes because `options.profile` is `undefined` and therefore `profile[0]` kills it.